sdk/nodejs/dagger/api/utils.ts

```diff
@@ -0,0 +1,42 @@
+  export function queryBuilder(q: any) {
```

Comment:
Is it volunteer to not type `q` for now?
Since it's type of `QueryTree`, you could just use it.

Reply:
It was volunteer when I was building the function (to make typescript happy). 
But it should be definitely typed. Thanks